frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +91,42 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
+      row {
+        radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            radioButtonComment
+          ),
+          OverrideOption.CUSTOM
+        )
+
+        val mutableList = branch.commitsUntilParent.toMutableList()
+        mutableList.remove(branch.forkPoint)
+        mutableList.remove(parent.pointedCommit)
+        val items = (listOf(branch.forkPoint, parent.pointedCommit) + mutableList.reversed()).filterNotNull()
```

Comment:
Whoops I'd rather keep the order of commits as-is, without putting inferred fork point & parent as first items... that's very confusing (I just tried to match the order from combobox to git history and could get it ;p)

Reply:
ok so to my understanding the most reasonable order is determined by these rules:
1. unique commits until parent in the same order as in git machete tree
2. fork point 
  - if fork point **is** present among the unique commits until parent then it remains in its place
  - if fork point **is not** present among the unique commits until parent then it is added BEFORE them
3. parent - the same rules as fork point (but applied later) 

@PawelLipski, is it correct?

Reply:
Okay, let's go with that 👍🏻